src/home_robot/home_robot/motion/stretch.py

```diff
@@ -215,6 +215,7 @@ class HelloStretchKinematics(Robot):
     look_at_ee = np.array([-np.pi / 2, -np.pi / 4])
     look_front = np.array([0.0, math.radians(-30)])
     look_ahead = np.array([0.0, 0.0])
+    max_arm_height = 1.2
```

Comment:
I have actually seen it go to heights that it finds hard to grasp at (for example the kitchen island in the fremont apartment). Does this height take into consideration that the grasping code that we have only does top down grasps?

Reply:
Yeah, this is the max possible arm extension. It can't go any higher than this.